recipes/forcebalance/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ForceBalance" %}
+{% set version = "1.7.3" %}
+{% set sha256 = "d662e48862c861af6b0bc1c917ef90c65a51ad7102368b0c3945c9aff3450621" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/leeping/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - scipy
+    - numpy
```

Comment:
scipy and numpy should be in host

Reply:
This is my workaround for needing them available when `setup.py` is invoked. I realize this is non-standard but is my best guess at how to accomplish that